src/soc/ibm/power9/romstage.c

```diff
@@ -340,6 +342,16 @@ void main(void)
 	uint8_t phb_active_mask = 0;
 	uint8_t iovalid_enable[MAX_PEC_PER_PROC] = { 0 };
 
+	if (ipmi_premem_init(CONFIG_BMC_BT_BASE, 0) != CB_SUCCESS)
+		die("Failed to initialize IPMI\n");
+
+	/*
+	 * Two minutes to load.
+	 * Not handling return code, because the function itself prints log messages
+	 * and its failure is not a critical error.
+	 */
+	(void)ipmi_init_and_start_bmc_wdt(CONFIG_BMC_BT_BASE, 120, TIMEOUT_HARD_RESET);
+
```

Comment:
Please move it below console_init(), maybe even init_timer() in case coreboot changes the way it implements timeouts in the future. Nothing is printed in this place, and I don't think die() would print anything, too.

Reply:
Moved after `init_timer()` (amended last commit).